app/queries/users/select_moderators_query.rb

```diff
@@ -1,9 +1,10 @@
-module Notifications
-  module Moderation
-    MODERATORS_AVAILABILITY_DELAY = 22.hours
-    SUPPORTED = [Comment].freeze
+module Users
+  class SelectModeratorsQuery
```

Comment:
This was a service worker and Joshua pointed out pretty accurately that `available_moderators` can be confusing and possibly misleading. The code in this service is a  User query (it returns users), but it's name spaced to `Notifications::Moderation`, however the service does not seem to do notification nor moderation. 

Hence, I thought about this more and decided to move the code into a User Query instead following some of the conventions we have in place. The previous version also housed two unrelated constants which I then moved into app/services/notifications/moderation/send.rb which seemed more related as it was being used in that file.